File: python/bee1080.py
cont = maior = pos = 0
while cont < 100:
    n = int(input())
    if cont == 0:
        maior = n
        pos = cont + 1
    else:
        if n > maior:
            maior = n
            pos = cont + 1
    cont += 1
print(maior)
print(pos)

File: c/bee1080.c
#include <stdio.h>
#define TAM 100
 
int main() {
    int numeros[TAM], maior, pos;
    for(int i=0; i<TAM; i++){
        scanf("%d", &numeros[i]);
    }
    for(int i=0; i<TAM; i++){
        if(i==0){
            maior = numeros[i];
            pos = i;
        } else{
            if(numeros[i]>maior){
                maior = numeros[i];
                pos = i;
            }
        }
    }
    printf("%d\n%d\n", maior, pos+1);
    return 0;
}